api/idea-card/models/idea-card.settings.json

```diff
@@ -60,6 +60,25 @@
         "Advanced"
       ],
       "required": true
+    },
+    "comments": {
+      "via": "idea_card",
+      "collection": "comment"
+    },
+    "discord": {
+      "type": "string"
+    },
+    "email": {
+      "type": "email"
+    },
+    "calendly": {
```

Comment:
Does every idea have a calendly link?

Reply:
That's correct!

Reply:
Can you explain a bit what the link is used for?

Reply:
That's a great question, I'm not sure about it. I'm going to ask the team and let you know.

Reply:
Hey chungthuang, it would be the project lead's Calendly. So people interested in joining the project can contact them about it.